ush/forecast_postdet.sh

```diff
@@ -784,7 +797,12 @@ MOM6_postdet() {
   $NCP -pf $FIXmom/$OCNRES/* $DATA/INPUT/
 
   # Copy coupled grid_spec
-  spec_file="$FIX_DIR/fix_cpl/a${CASE}o${OCNRES}/grid_spec.nc"
+  if [ $APP = "NG-GODAS" ];then
```

Comment:
This shouldn't be necessary. `data_out_GFS()` is only called if `RUN` is `gfs`. `data_out_DATM()` gets called when `RUN` is `datm` (which I believe it should be if the `APP` is `NG-GODAS`.